src/style.css

```diff
@@ -0,0 +1,56 @@
+* {
+  margin: 0;
+}
+
+.wrap {
+  padding: 32px 22px;
+}
```

Comment:
Do you need this padding here?
Look at your tests, you have a problem because of it
![image](https://user-images.githubusercontent.com/57747481/173309591-ed67a88f-98b9-446a-b5cc-d72c8ae2345e.png)